### PR TITLE
eoan: Add persistent GPU offloading

### PR DESCRIPTION
**Pull request by crawfxrd**
When `X-KDE-RunOnDiscreteGpu` is set in the application's `.desktop` file, launch the application on the discrete GPU if available, and provide the option to launch on the integrated GPU.

Requires: #13

### CODE REVIEW

**Review by jackpot51 — APPROVED** [review]

**Review by jackpot51 — APPROVED** [review]

**Review by brs17 — APPROVED** [review]
`X-KDE-RunOnDiscreteGpu` seems to be respected and used as desired. When testing this pr I noticed an issue with https://github.com/pop-os/gnome-shell/pull/13. This pr is good to merge once #13 is approved.

**Comment by crawfxrd** [comment]
Will not be added to 19.10. See #23 for implementation in 20.04.

**Pull request closed without merging**
